source/_integrations/sonos.markdown

```diff
@@ -40,6 +40,50 @@ The battery sensors rely on working change events or updates will be delayed. S1
 
 The Sonos integration adds one `switch` for each alarm set in the Sonos app. The alarm switches are detected, deleted and assigned automatically and come with several attributes that help to monitor Sonos alarms.
 
+## Playing media
+
+Sonos accepts a variety of `media_content_id` formats in the `media_player.play_media` service, but most commonly as URIs. For example, both Spotify and Tidal share links can be provided as-is. Playback of [music hosted on a Plex server](/integrations/plex#sonos-playback) is possible. Direct HTTP/HTTPS links to local or remote media files can also be used if the Sonos device can reach the URI directly, but specific media encoding support may vary.
+
+Music services which require an account (e.g., Spotify) must first be configured using the Sonos app.
+
+An optional `enqueue` argument can be added to the service call. If `true`, the media will be appended to the end of the playback queue. If not provided or `false` then the queue will be replaced.
+
+### Examples:
+
+This is an example service call that plays an audio file from a web server on the local network (like the Home Assistant built-in webserver):
+
+```yaml
+service: media_player.play_media
+target:
+  entity_id: media_player.sonos
+data:
+  media_content_type: "music"
+  media_content_id: "http://192.168.1.50:8123/local/sound_files/doorbell-front.mp3"
+```
+
+Sonos can also play music or playlists from Spotify. Full Spotify URIs and ID codes can both be used directly. An example service call:
```

Comment:
This is incorrect now. Spotify ID codes are not usable, the example shows a Spotify URI. For more information, see the Spotify developers documentaion.

Reply:
Wrong terminology. Looks like I mean URIs and URLs, but not sure if that will be obvious to users. Maybe a second Spotify example would help.